docs/smtchecker.rst

```diff
@@ -457,6 +457,7 @@ list of one or more verification targets. The keywords that represent the target
 - Division by zero: ``divByZero``.
 - Trivial conditions and unreachable code: ``constantCondition``.
 - Popping an empty array: ``popEmptyArray``.
+- Out of bounds array/fixed bytes index access: ``outOfBounds``.
```

Comment:
There is another place where you mention verification targets around line 30, maybe mentioning out of bounds there as well? For example array pop on empty array is there.